meilisearch-http/src/option.rs

```diff
@@ -17,9 +17,6 @@ pub struct Opt {
     pub master_key: Option<String>,
 
     /// This environment variable must be set to `production` if your are running in production.
-    /// Could be `production` or `development`
-    /// - `production`: Force api keys
-    /// - `development`: Show logs in "info" mode + not mendatory to specify the api keys
```

Comment:
> If the development flag is set it will show more logs, and the master key can be avoided which implies that there is no security on the updates routes. This is useful to debug when integrating the engine in another service.

Reply:
I fix it: 
<img width="853" alt="Capture d’écran 2020-03-06 à 10 16 40" src="https://user-images.githubusercontent.com/6064892/76069706-93a33780-5f93-11ea-8617-f4a7b2309e87.png">